superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -481,6 +497,27 @@ export default class ResultSet extends React.PureComponent<
     return <div className="noControls" />;
   }
 
+  rowsReturned() {
+    const { query } = this.props;
+    const limitWarning = <Icon className="ReturnedRowsImage" name="warning" />;
+    return (
+      <ReturnedRows>
+        {query.results?.displayLimitReached && limitWarning}
+        <span>{t(`%s rows returned`, query.rows)}</span>
+        {query.results?.displayLimitReached && (
```

Comment:
lets see if we can put this value into a variable since you're using it twice. You can also declare it in the destructuring on line 501.